packages/core-backend/src/BackendWebSocketService.test.ts

```diff
@@ -889,7 +889,7 @@ describe('BackendWebSocketService', () => {
           service.connect();
 
           // Advance time past the timeout
-          await completeAsyncOperations(110);
+          await completeAsyncOperations(101);
```

Comment:
Nice! How come this fixes it? We wait 101ms instead of 110ms? Does the 9ms make a large difference?